lib/fastlane/actions/update_app_identifier.rb

```diff
@@ -19,7 +20,9 @@ def self.run(params)
           project = Xcodeproj::Project.open(project_path)
 
           # Fetch the build configuration objects
-          configs = project.objects.select { |obj| obj.isa == 'XCBuildConfiguration' && !obj.build_settings[identifier_key].nil? }
+          configs = project.objects.select { |obj| obj.isa == 'XCBuildConfiguration' &&
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.